sdks/go/pkg/beam/io/textio/sdf.go

```diff
@@ -37,17 +37,21 @@ func init() {
 
 // ReadSdf is a variation of Read implemented via SplittableDoFn. This should
 // result in increased performance with runners that support splitting.
+//
+// Deprecated: Called directly by Read, use that instead.
 func ReadSdf(s beam.Scope, glob string) beam.PCollection {
-	s = s.Scope("textio.ReadSdf")
+	s = s.Scope("textio.Read")
 
 	filesystem.ValidateScheme(glob)
 	return readSdf(s, beam.Create(s, glob))
 }
 
 // ReadAllSdf is a variation of ReadAll implemented via SplittableDoFn. This
 // should result in increased performance with runners that support splitting.
+//
+// Deprecated: Called directly by ReadAll, use that instead.
 func ReadAllSdf(s beam.Scope, col beam.PCollection) beam.PCollection {
-	s = s.Scope("textio.ReadAllSdf")
+	s = s.Scope("textio.ReadAll")
```

Comment:
Thoughts on moving this file's contents into the main `textio.go` file? If we're removing the distinction between read and readSdf, splitting doesn't make sense anymore IMO

Reply:
I agree, no reason to logically split it out if it's SDFs all the way down